cypress/integration/explore.spec.js

```diff
@@ -138,4 +138,53 @@ context('Actions', () => {
       `[data-testid=resource-type-filtering-checkbox-${ResourceFeatureTypes.video.toString().toLowerCase()}] input`
     ).should('be.checked');
   });
+
+  it('adds and remove tags as a filters', () => {
```

Comment:
foretrakk å trekke sammen noen av metodene som jeg tenkte hørte sammen, for å slippe for mye dobbelt opp

Reply:
Den er grei